src/plot_api/plot_api.js

```diff
@@ -2156,6 +2157,82 @@ exports.update = function update(gd, traceUpdate, layoutUpdate, _traces) {
     });
 };
 
+/**
+ * Convert a primitive TypedArray representation object into a TypedArray
+ * @param {object} v: Object with `dtype` and `data` properties that
+ * represens a TypedArray.
+ *
+ * @returns {TypedArray}
+ */
+var dtypeStringToTypedarrayType = {
```

Comment:
We'll need to append this list:

https://github.com/plotly/plotly.js/blob/a8c621706326dafbfe385418874a54c4723cbc4e/.eslintrc#L12-L21

to make `npm run lint` pass.